addons/l10n_fr_pos_cert/models/pos.py

```diff
@@ -0,0 +1,194 @@
+from datetime import datetime
+from hashlib import sha256
+from json import dumps
+import pytz
+
+from openerp import models, api, fields
+from openerp.fields import Datetime
+from openerp.tools.translate import _
+from openerp.exceptions import UserError
+
+NOT_SAME_DAY_ERROR = _("This session has been opened another day. To comply with the French law, you should close sessions on a daily basis. Please close session %s and open a new one.")
+
+
+def ctx_tz(record, field):
```

Comment:
This looks overly complicated on top of `fields.Datetime.context_timestamp`. Not sure it's necessary?

Reply:
I am afraid it is (considering the current crappy implementation of the reporting and user friendliness of the output): context_timestamp doesn't format the date to the locale